README.md

```diff
@@ -114,6 +116,17 @@ The gulpfile has a number of useful tasks which can be run with "gulp taskName",
  * down: stops all the containers and deletes them
 
 
+## How add additional controllers
+  In order to extend the educandu functionality you can add your custom controllers. These should be classes that expose one or more of the following methods:
+   * registerApi: if you wish to register some API calls
```

Comment:
Rather `.. register API endpoints` here

Reply:
Yes and there's another typo on the next line, thanks!

Reply:
Fixed